feat(issue_templates): Add issue templates for bug, features, and game data (#52)
Prophecy-eip/Backend: .github/ISSUE_TEMPLATE/feature.md

---
name: Feature request
about: Request a new feature
title: "[FEAT] Description"
labels: enhancement
assignees:

---

> **Before requesting a feature, please check that someone did not request the same feature as yours**

Author: ```suggestion
**Before requesting a feature, please check that someone did not request a similar one**
```

## Feature emplacement
- [ ] Server
- [ ] API
- [ ] Database
- [ ] Other:

> ## What is expected from the feature

Author: ```suggestion
## Detailed description
*What is expected from the feature*

## What currently exists (if relevant)

```
